### ISSUE TITLE
Document structure seems odd to me.

### ISSUE THREAD

**Issue by jenstroeger** [IDPF/epub-vocabs#9]
Looking at [Document reference sections](https://idpf.github.io/epub-vocabs/structure/#references) it’s followed by three unnumbered sections of the _same_ visual level: [Indexes](https://idpf.github.io/epub-vocabs/structure/#indexes), [Glossaries](https://idpf.github.io/epub-vocabs/structure/#glossaries), [Bibliographies](https://idpf.github.io/epub-vocabs/structure/#bibliographies). Looking at the DOM, though, those three seem to be sub-sections:

![structure](https://user-images.githubusercontent.com/12053937/77817145-56564380-7114-11ea-95fb-3bd584bba038.jpg)

However, they’re using `H2` headers, and yet aren’t listed in the page’s TOC alongside other `H2` headings. I think they should either be `H3` or should be equally listed in the TOC; I lean towards to former because the three are indeed reference sections.

**Comment by mattgarrish** [comment]
> I think they should either be `H3`

Yes, and the divs should really also be sections. The document used to be dynamically generated from an n3 file and we just took a static snapshot of the output when we moved away from that process. I wouldn't be surprised if there are other oddities lurking in it.

**Comment by jenstroeger** [comment]
So, what’s the plan with these documents now?

Maintain the HTML or move to some other meta format, e.g. Markdown or, considering the documentation is already hosted on Github, to Wiki or some such?

**Comment by mattgarrish** [comment]
Couldn't say at this time. Will have to see what taking epub to rec in w3c entails. My guess would be it gets re-done in respec without all the RDF vocab stuff.
